### ISSUE TITLE
Transport Types -> BLE mentions attachments inaccurately

### ISSUE THREAD

**Issue by okdistribute** [getditto/alphadocs#3]
**Describe the issue**

In the Transport Types page under BLE, the page says

'When replicating large amount of data, use an attachment..."
 
<img width="840" alt="Screenshot 2023-08-03 at 4 21 14 PM" src="https://github.com/getditto/alphadocs/assets/633012/819985b8-6fac-4c55-841e-a7c3983bf160">

However, it is not recommended to syncronize large objects, even attachments, if the primary method of communication is going to be bluetooth. 

**To Reproduce**
https://alphadocs.ditto.live/transport-types#wiVpR

**Expected behavior**

Instead, it would be more helpful to link to a page that discusses flat data models, since this will allow us to parallelize over the bluetooth connection. See  https://docs.ditto.live/android/common/concepts/schemas#%EF%B8%8F-bad-pattern-big-documents

**Screenshots**
**Additional context**

**Comment by skylerjokiel** [comment]
I'm not sure I understand why this is inaccurate or not-recommended.

My understanding is that Attachments are lazy loaded on the device and only replicate when the Attachment handle is accessed. So I read the statement more as a generic reminder to store large data using attachments.

BLE, in particular, is bandwidth constrained so if you have large data + BLE then you should definitely use attachments as opposed to storing your large data sets in the document where it's guaranteed to be replicated.

Can you elaborate on the recommended way for customers with large data that may be using ble?

**Comment by okdistribute** [comment]
Yeah please see this section which has the relevant clauses about Bluetooth:

https://docs.ditto.live/android/common/concepts/schemas#%EF%B8%8F-bad-pattern-big-documents

Your document also needs to define "large" - over 250kb is not recommended in a single document. 

Attachments can be arbitrarily large, but are not recommended for use cases where Bluetooth is the only available transport. Using attachments is recommended only when there is a homogenous mesh (all iOS or all Android), and mixed meshes need a LAN or Big Peer connection. Additionally, Bluetooth bandwidth can get clogged by attachments so it is not recommended to sync attachments over Bluetooth.

Because attachments are retrieved on demand, this can be monitored controlled by the developer. In a private setting I'm happy to walk you through customer use cases for fetching attachments today in production.
